ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -63,7 +63,15 @@ import RevenueCat
     }
 
     @objc public static func setAutomaticAppleSearchAdsAttributionCollection(_ enabled: Bool) {
-        Purchases.automaticAppleSearchAdsAttributionCollection = enabled
+        // Using `setValue` because `automaticAppleSearchAdsAttributionCollection` is now deprecated and we get a warning
+        Purchases.setValue(enabled, forKey: "automaticAppleSearchAdsAttributionCollection")
```

Comment:
![CatBobbing](https://user-images.githubusercontent.com/1304821/183542886-d8e662bf-0b70-4248-b038-94b910aa212c.gif)

Reply:
⚠️  this seems super fragile.
Why don't we mark `setAutomaticAppleSearchAdsAttributionCollection` as deprecated as well? That way we don't need the hack, and we propagate the deprecation.

Reply:
#199